src/main/scala/beam/agentsim/scheduler/TriggerMeasurer.scala

```diff
@@ -14,102 +22,178 @@ import scala.collection.mutable.ArrayBuffer
   This is dumb implementation. We store all the data to build percentiles. There are different approaches to get percentile on stream
   More: https://blog.superfeedr.com/streaming-percentiles/
  */
-class TriggerMeasurer extends LazyLogging {
+class TriggerMeasurer(val cfg: BeamConfig.Beam.Debug.TriggerMeasurer) extends LazyLogging {
```

Comment:
Same - this makes #1180 pointless

Reply:
I merged Art's branch into my own because it had useful changes for my debugging.